pymc3/distributions/continuous.py

```diff
@@ -1776,8 +1776,15 @@ def logp(self, value):
             0 < self.kappa,
         )
 
+class Lognormal(LogNormal):
+    """deprecated, use LogNormal instead!"""
+    def __init__(self, *args, **kwargs):
+#         warnings.warn(
```

Comment:
I agree that we should just silently allow this for backwards compat. But then the comment can just be removed.

Reply:
im not sure of how this gets printed in the documentation, but if it does wouldnt thecomment make sense for users to to see this in the documentation?

Reply:
The docstring, yes, but I mean the commented warning.